app/views/api/v1/instances/show.rabl

```diff
@@ -1,6 +1,12 @@
 object false
 
-node(:uri)         { site_hostname }
-node(:title)       { Setting.site_title }
-node(:description) { Setting.site_description }
-node(:email)       { Setting.site_contact_email }
+node(:uri)                  { site_hostname }
+node(:title)                { Setting.site_title }
+node(:description)          { Setting.site_description }
```

Comment:
For this one and the next few lines, I believe InstancePresenter knows about all of these values, and any that it doesn't could be added.